how-to-contribute-to-workshops/README.md

```diff
@@ -0,0 +1,28 @@
+# Workshops
+
+## Purpose
+
+Bit Project implements workshops in order to help students, regardless of their level of technical knowledge, develop pertinent skills and learn new technologies. 
+
+## Workshop Format
+
+These workshops are typically created in the form of slides/ presentations along with an interactive activity in which students can apply the knowledge that they gained throughout the presentation. Note that this interactive activity is **NOT** of the form of a "quiz" game, but rather, is an activity which allows students to critically think about and apply what they have learned (See [How to Contribute to Workshops](./How-to-contribute-to-workshops.md) for more details). 
+
```

Comment:
I think the link here is not very necessary, since you did not talk about what activities are in the how-to-contribute-to-workshops.md. Maybe link an example of a workshop slides instead?